Utils/Dataflow/016_task2es/task2es.py

```diff
@@ -183,28 +183,65 @@ def input_events(data):
     return result
 
 
+def transform_chain_data(data):
+    """ Transform chain_data into array of integers and get chain_id from it.
+
+    chain_id is the taskid of the task chain's root.
+
+    :param data: data to be updated, must contain taskid and proper chain_data
+                 (string of numbers separated by commas).
+    :type data: dict
+
+    :return: True if update was successful, False otherwise (chain_id and
+             chain_data will be updated so that the task is considered
+             independent)
+    :rtype: bool
+    """
+    if type(data) is not dict:
+        return False
```

Comment:
Would be nice to have some notification here.

Reply:
Done.